### ISSUE TITLE
Armor doesn't work on multiplayer, and the Advancements tab doesn't show up

### ISSUE THREAD

**Issue by CammiePone** [rwtema/Monk-Mod#41]
**Monk Mod Version: **
1.3.3

**Mod List (it's long, bear with me): **
The Betweenlands
Aether Legacy
Applied Energistics 2
Arche
Astral Sorcery
AuroRegLib
Backpacks!
Baubles
BetterFPS
Blood Magic
Botania
Chisel
Chisels and Bits
CodeChicken Lib
CoFH Core
CoFH World
ConnectedTexturesMod
Construct's Armory
Cosmetic Armor Reworked
Custom NPCs
Cyclops Core
DummyCore
Embers
EssentialCraft 4
EvilCraft
Extra Utilities
Extreme Reactors
Fast Leaf Decay
FoamFix
Guide-API
Hammer Core
Hwyla
Immersive Engineering
Inventory Tweaks
Iron Chests
JourneyMap
JEI
Mantle
Matter Overdrive
Mekanism
Mekanism Generators
Mekanism Tools
More Overlays
Nether Chest
Psi
ReAuth
Reborn Core
Redstone Flux
Rustic
Simple Storage Network
Simply Jetpacks 2
Solar Flux Reborn
Steve's Carts Reborn
Thaumcraft
The Erebus
The Monk Mod (obviously :p)
Thermal Dynamics
Thermal Expansion
Thermal Foundation
Tinkers Aether
Tinkers Construct
Tinkers' Tool Leveling
Totemic
VeinMiner
Voidaic Arcania
Waila Harvestability
Waystones
ZeroCore

**Exactly what's happening: **
The armour bars display, however, you take the same amount of damage regardless of if they're there after level 20, or if you haven't even hit level 1.

As for the advancements tab not showing, it's just not showing. Advancements also don't appear in chat, despite that being on. Everything else works fine, though.

**Comment by alexsch01** [comment]
direwolf20 on forgecraft has no issues with armor so prob start a new world

**Comment by CammiePone** [comment]
> direwolf20 on forgecraft has no issues with armor so prob start a new world

We're clearly not running ForgeCraft, and the world was started with the Monk Mod. I'm 99% sure it's some kind of bug, which bugs should be reported, regardless of whether or not it can be quickly fixed for us in this moment. 

Also, it took 3 days for 4 people to build spawn, I really don't feel like fully restarting the world. I just wanted to inform rwtema of this bug, so that for future releases it could hopefully be fixed.

**Comment by CammiePone** [comment]
Okay, so I'm currently restarting the world, as painful as it'll be, because updating to 1.4 of the mod didn't fix the problem. So we'll see what happens in roughly... 20 minutes.

**Comment by CammiePone** [comment]
Nope, restarting the world does not fix the problem. It is indeed a bug that is not fixable by me.

**Comment by Reploidsham** [comment]
I'm having armour stats not being functional as well but I'm using a self-built modpack and there's no logs indicating any error. The advancement tab not showing can be fixed by punching ten VANILLA trees and then dying. It's brutal, but that seems to fix it for me.
